deepchem/utils/molecule_graph.py

```diff
@@ -0,0 +1,140 @@
+from typing import Optional, Iterable
+import numpy as np
+
+
+class MoleculeGraphData(object):
```

Comment:
Could you add this class to `docs/dataclasses.rst`?

Reply:
👍